tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -281,14 +280,14 @@ private void buildTensorFlowSession() {
     String taskCommand = "'" + baseTaskCommand + "'";
     LOG.info("Final task command: " + taskCommand);
 
-    TensorFlowSession.Builder builder = new TensorFlowSession.Builder()
+    TonySession.Builder builder = new TonySession.Builder()
         .setTaskCmd(taskCommand)
         .setVenv(pythonVenvZip)
         .setAMAddress(amHostPort)
         .setShellEnv(shellEnv)
+        .setTonyConf(tonyConf)
         .setTaskExecutorJVMArgs(tonyConf.get(TonyConfigurationKeys.TASK_EXECUTOR_JVM_OPTS,
```

Comment:
since we're passing tonyConf into the builder perhaps we can remove this and extract the jvm_opts config there?

Reply:
oh yah, missed to clean that part. We can merge this clean up with another PR.